service/docs/source/admin.rst

```diff
@@ -16,42 +15,45 @@ Service.  Additional information is available on other pages:
 Linux Integration
 -----------------
 
-The GEOPM Service integrates with the Linux OS through Systemd as a
-unit that is installed with the geopm-service RPM.  The ``sytemctl``
-command can be used to interact with the ``geopm`` Systemd Unit.
+The GEOPM Service integrates seamlessly with the Linux OS through Systemd. It
+is packaged within the geopm-service binary package, and administrators can install it
+using their respective package management systems. Use ``systemctl``
+to interact with ``geopm`` Systemd Unit.
 
 
 GEOPM Service Files
 -------------------
 
-In addition to the files provided by the installation packages, the
-GEOPM Service may create and modify files while active.  The files
-created by the GEOPM Service are located within two directories.  The
-files in ``/etc/geopm`` hold the access control lists.  These
-files persist across reboots and restarts of the service.  The files
-in ``/run/geopm`` track information about clients that are
-actively using the service.  These files save the state of the GEOPM
-Service and if the service is stopped for any reason the files will be
-used when the service is started again.  The files in ``/run`` are
-erased upon reboot.
-
-All files and directories within ``/etc/geopm`` or
-``/run/geopm`` are created by the GEOPM Service with
-restricted access permissions and root ownership.  The GEOPM Service
-will not read any file or directory if they are modified to have more
-permissive access restrictions, non-root ownership, or if they are
-replaced by a symbolic link or other non-regular file.  If these
-checks fail, the file or directory will be renamed to include a UUID
-string and a warning is printed in the syslog.  These renamed files or
-directories enable an administrator to perform an investigation into
-problem, but they will not be used by the GEOPM Service in any way.
-
-It is recommended that these GEOPM Service system files are always
-manipulated using GEOPM tools like ``geopmaccess``, however, any
-administrator that manipulates the GEOPM system files without using a
-GEOPM interface should be aware of the permission and ownership
-requirements for these files.  For more information about the GEOPM
-security model please refer to the `Security Guide <security.html>`_.
+Beyond the files that come with the installation packages, the GEOPM Service
+may generate and modify additional files during its active state. These files
+are housed within two primary directories:
+
+- ``/etc/geopm``: This directory contains access control lists. Files here
+  persist across both reboots and service restarts.
```

Comment:
More generally, it contains any config files. Other than access control config, MSRIOGroup configs and ConstConfigIOGroup configs come to mind. The service does come with MSRIOGroup configs preinstalled, but additional ones could be configured by an admin.